From 0b62f71eb5d44d4a57103566ba58022b6304fa4f Mon Sep 17 00:00:00 2001
From: Tom Rini <tom_rini@mentor.com>
Date: Wed, 27 Jul 2011 03:46:52 +0000
Subject: [PATCH] rp-pppoe: Port from oe.dev

The autoconf stuff is all in a subdirectory, which is rather annoying
as OE expects patches to be applied and autoconf stuff to be done in
S. This adds enough autoconf at the top level to allow it to be
called there - all it does is run a sub autoconf stuff in the src
directory.

Upstream-Status: Inappropriate [build system specific change]

---
 Makefile.am  | 5 +++++
 configure.in | 4 ++++
 2 files changed, 9 insertions(+)
 create mode 100644 Makefile.am
 create mode 100644 configure.in

diff --git a/Makefile.am b/Makefile.am
new file mode 100644
index 0000000..eff6977
--- /dev/null
+++ b/Makefile.am
@@ -0,0 +1,5 @@
+PACKAGE = rp-pppoe
+VERSION = 3.8
+
+dnl AM_CFLAGS = -Wall -DDEBUG
+SUBDIRS = src
diff --git a/configure.in b/configure.in
new file mode 100644
index 0000000..ac1a7e4
--- /dev/null
+++ b/configure.in
@@ -0,0 +1,4 @@
+AC_INIT(src/pppoe.c)
+AM_INIT_AUTOMAKE([rp-pppoe], [3.8])
+AC_CONFIG_SUBDIRS(src)
+AC_OUTPUT(Makefile)
